mnefun/_mnefun.py

```diff
@@ -1727,16 +1728,19 @@ def gen_forwards(p, subjects, structurals, run_indices):
                                 subj + '-trans_head2mri.txt')
                 if not op.isfile(trans):
                     raise IOError('Unable to find head<->MRI trans file')
-            src_space_file = op.join(subjects_dir, structurals[si], 'bem',
-                                     structurals[si] + '-oct6-src.fif')
-            if not op.isfile(src_space_file):
+            for mid in ('oct6', 'oct-6'):
+                src_space_file = op.join(subjects_dir, struc, 'bem',
+                                         '%s-%s-src.fif' % (struc, mid))
```

Comment:
Isn't just cleaner if `src_space_files ` are named as expected i.e., `np-src`, renaming can be done with a simple bash command. Or is this addressing some other issue IDK about?

Reply:
Yeah it would be cleaner. I indeed added this to address a corner case where @mdclarke had a morphed structural that used the `npsrc` instead of `np-src` style naming. I structured this to continue to prefer our `np-src`-style naming, though, so I think we should be okay